src/main/java/util/csvparsing/CsvFileHelper.java

```diff
@@ -0,0 +1,19 @@
+package util.csvparsing;
```

Comment:
Renamed this to be `CsvFileHelper`. No need to instantiate it right now as the only method it has is `static`.